src/region/regionUtil.ts

```diff
@@ -1,5 +1,15 @@
 import * as React from 'react';
 import { Annotation, AnnotationRegion, Position } from '../@types';
+import { invertYCoordinate, Point, rotatePoint, translatePoint } from './transformUtils';
```

Comment:
Should this be `transformUtil` to stay consistent with `regionUtil`? Or can they just be combined into the latter?